.github/workflows/docker.yml

```diff
@@ -0,0 +1,88 @@
+---
+
+###
+### Build Docker image
+###
+name: docker
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    name: docker
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+
+      - name: Set variables
+        run: |
+          # This can be the branch name (master, release-) or a tag name
+          TAG="$( echo "${GITHUB_REF}" | cut -d / -f 3 )"
+          if [ "${TAG}" = "master" ]; then
+            TAG="latest"
+          fi
+          echo "TAG=${TAG}"
+          echo "TAG=${TAG}" >> ${GITHUB_ENV}
+
+      # ------------------------------------------------------------
+      # Build
+      # ------------------------------------------------------------
+      - name: Build Docker image
```

Comment:
Why do we need a retry? 🤔

Reply:
I think we had this discussion a few months ago.

This is from my personal experience with building Docker images (I have more than 300 images build each day from my own repositories). Every now and then GitHub actions had a tiny network outage which broke some of my builds on a regular base (when they do stuff like `apt update` or `apt install` or `apk add` etc).

The whole retry thing fixed this entirely

Reply:
Thanks for explanation!